doc/shield-coinbase.md

```diff
@@ -0,0 +1,101 @@
+# Shielding Coinbase UTXOs
+
+**Summary**
+
+Use z_shieldcoinbase RPC call to shield coinbase UTXOs.
+
+**Who should read this document**
+
+Miners, Mining pools, Online wallets
+
+## Background
+
+The current Zcash protocol includes a consensus rule that coinbase rewards must be sent to a shielded address.
+
+## User Experience Challenges
+
+A user can use the z_sendmany RPC call to shield coinbase funds, but the call was not designed for sweeping up many utxos, and offered a suboptimal user experience.
```

Comment:
```
`z_sendmany`
```

`UTXOs`

Reply:
ok, hmm, thought i did this already with search and replace.